### PR TITLE
Add label area:providers to boring cyborg

### LINKED ISSUE #14879: Add area:providers for general providers to boring-cyborg
**Description**

Add label **area-providers** to general providers.
For example the **provider:Google**
 https://github.com/apache/airflow/blob/16f43605f3370f20611ba9e08b568ff8a7cd433d/.github/boring-cyborg.yml#L21-L25

**Use case / motivation**

Help better issue/pull request monitoring.

**Are you willing to submit a PR?**

Yes

**Related Issues**

### PR DESCRIPTION
**Pull request by marcosmarxm**
Boring Cyborg now only allows labeling using a basic path. When someone changes a big provider (Apache, Google, AWS) the boring cyborg is going to apply two labels: `area:providers` and `provider:Google` for example, which is not a problem IMHO.

Resolves #14879 @kaxil and @mik-laj can take a look?

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

File: .github/boring-cyborg.yml [hunk 1]
```diff
@@ -44,6 +44,15 @@ labelPRBasedOnFilePath:
...
     - tests/providers/snowflake/**/*
 
+  area:providers:
+    - airflow/providers/**/*
+    - docs/apache-airflow-providers-*/**/*
+    - tests/providers/**/*
+    - scripts/docker-compose/ci/integration-*.yml
+    - tests/core/test_providers_manager.py
+    - scripts/in_container/run_install_and_test_provider_packages.sh
+    - scripts/in_container/run_prepare_provider_documentation.sh
```
Comment (kaxil) [inline .github/boring-cyborg.yml]: Let's remove them, not strictly needed

```suggestion
```

File: airflow/providers/airbyte/hooks/airbyte.py [hunk 2]
```diff
@@ -93,6 +93,7 @@ def submit_sync_connection(self, connection_id: str) -> Any:
             endpoint=f"api/{self.api_version}/connections/sync",
             json={"connectionId": connection_id},
             headers={"accept": "application/json"},
+            extra_options={'allow_redirects': False}
```
Comment (kaxil) [inline airflow/providers/airbyte/hooks/airbyte.py]: unrelated change?

Comment (marcosmarxm) [reply airflow/providers/airbyte/hooks/airbyte.py]: Yes!

File: .github/boring-cyborg.yml (hunk 1 above)
Comment (marcosmarxm) [reply .github/boring-cyborg.yml]: Ok, I tried to add common files changed when adding/update providers.

Comment (marcosmarxm) [reply .github/boring-cyborg.yml]: @kaxil done =)

**Review by kaxil — APPROVED** [review]

**Comment by kaxil** [comment]
Thanks @marcosmarxm
